Assets/Scripts/Board/Board.cs

```diff
@@ -79,6 +79,42 @@ public TerrainTile(Vector3Int position, TileType tileType, int diceNumber = 0)
 
     private List<BoardTile> boardTiles = new List<BoardTile>();
 
+    // Graph
+    private class Node { }
+
+    private class VertexNode : Node
+    {
+        private EdgeNode[] edgeNodes;
+
+        public VertexNode() { }
+    }
+
+    private class EdgeNode : Node
+    {
+        private VertexNode[] vertexNodes;
+
+        public EdgeNode() { }
+    }
+
+    private class TileNode : Node
+    {
+        private VertexNode[] vertexNodes;
+
+        public TileNode() { }
+    }
+
+    private class BoardGraph
+    {
+        private VertexNode[] vertexNodes;
+        private EdgeNode[] edgeNodes;
+        private TileNode[] tileNodes;
+
+        private BoardGraph() { }
+
+
+    }
```

Comment:
Do you think we should just store these lists as members of the `Board` class rather than in this separate subclass? Not opposed to either initially
also nit: whitespace

Reply:
That feels fine to me. Fixed.